app/views/posts/show.html.erb

```diff
@@ -1,20 +1,19 @@
 <div>
-  <section>
-    <div>
-      <h1>Post: <%= @post.title %> <span>by <%=@user.name%></span></h1>
-      <span>Comments: <%= @post.comments_counter %></span>
-      <span>Likes: <%= @post.likes_counter %></span>
+  <section class="comments">
+    <div class="p-lk-cm">
+      <h3>Post <%='#'%><%= @post.id %> by <%= @user.name %> </h3>
+      <span>Comments: <%= @post.comments_counter %> Likes: <%= @post.likes_counter %></span>
     </div>
     <p><%= @post.text %></p>
   </section>
 
-  <section>
-  <ul>
-  <% @post.comments.each do |comment| %>
-    <li>
-      <span><%= comment.author_id %>: <%= comment.text %></span>
-    </li>
-  <% end %>
-</ul>
+  <section class="comments">
+    <ul class="clean-ul">
+    <% @post.comments.each do |comment| %>
+      <li>
+        <span><%= comment.author.name %>: <%= comment.text %></span>
+      </li>
+    <% end %>
```

Comment:
- [ ] Kindly make sure the Post show page displays the last five comments on each post as shown on this [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_single_post.png) for the project requirement